### ISSUE TITLE
ShareGPT4V abnormal performance

### ISSUE THREAD

**Issue by Kuangdd01** [ShareGPT4Omni/ShareGPT4V#19]
Thank you to the authors for providing the valuable open-source data. During our research, we re-evaluated [ShareGPT4V](https://huggingface.co/Lin-Chen/ShareGPT4V-7B) using the `share4v.eval` code from the repository. However, the results we obtained were significantly lower than those reported in the paper. The data table is shown below.

Model | VQAv2 | GQA | VisWiz | SciQA | TextVQA
-- | -- | -- | -- | -- | --
ShareGPT4V-7B* | 80.6 | - | 57.2 | 68.4 | -
ShareGPT4V-Reproduce | - | 61.41 | 53.02 | 67.7 | 57.57

Model | MME-P | MME-R | MMB | MMB-CN | SEED-IMG 
-- | -- | -- | -- | -- | --
ShareGPT4V-7B* |1567.4 | 376.4 | 68.8 | 62.2 | 69.7
ShareGPT4V-Reproduce |1494.2 | 335.35 | 66.5 | 58.4   | 64.79

Our evaluation environment used an A800 GPU.

The Python environment is as follows: 
```
python                    3.10
accelerate                0.21.0
numpy                     1.26.4
nvidia-cublas-cu12        12.1.3.1
nvidia-cuda-cupti-cu12    12.1.105
nvidia-cuda-nvrtc-cu12    12.1.105
nvidia-cuda-runtime-cu12  12.1.105
nvidia-cudnn-cu12         8.9.2.26
nvidia-cufft-cu12         11.0.2.54
nvidia-curand-cu12        10.3.2.106
nvidia-cusolver-cu12      11.4.5.107
nvidia-cusparse-cu12      12.1.0.106
nvidia-ml-py              12.560.30
nvidia-nccl-cu12          2.18.1
nvidia-nvjitlink-cu12     12.6.85
nvidia-nvtx-cu12          12.1.105
torch                     2.1.2
torchvision               0.16.2
tqdm                      4.67.1
transformers              4.37.2
```

**If I have missed anything, please kindly point it out. I would greatly appreciate it.**  🙇🙇

### Q1:  What could be the potential reasons for the significant decrease in evaluation results?

In addition, I am also interested in a set of experimental details from the paper.
### Q2: For Table 4 in the provided data, was the vision encoder trained during the pre-training and SFT training processes?

![Image](https://github.com/user-attachments/assets/544302be-6dac-4e32-9e1c-f476ae839f66)

Lastly, I would like to thank the authors for their valuable work!  😊
